scripts/globals/weaponskills/swift_blade.lua

```diff
@@ -18,23 +18,28 @@ require("scripts/globals/weaponskills")
 local weaponskill_object = {}
 
 weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary, action, taChar)
-
     local params = {}
     params.numHits = 3
     params.ftp100 = 1.5 params.ftp200 = 1.5 params.ftp300 = 1.5
-    params.str_wsc = 0.3 params.dex_wsc = 0.0 params.vit_wsc = 0.0 params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.3 params.chr_wsc = 0.0
+    params.str_wsc = 0.3 params.dex_wsc = 0.0 params.vit_wsc = 0.0
+    params.agi_wsc = 0.0 params.int_wsc = 0.0 params.mnd_wsc = 0.3
+    params.chr_wsc = 0.0
     params.crit100 = 0.0 params.crit200 = 0.0 params.crit300 = 0.0
     params.canCrit = false
-    params.acc100 = 0.8 params.acc200= 0.9 params.acc300= 1
-    params.atk100 = 1; params.atk200 = 1; params.atk300 = 1
+    -- Sufficient data for ACC bonus/penalty does not exist; assuming no penalty and 10% increase per 1,000 TP
+    -- http://wiki.ffo.jp/html/382.html does not list ACC Bonus
+    -- https://www.bg-wiki.com/ffxi/Swift_Blade does not list ACC Bonus
+    params.acc100 = 1.0 params.acc200 = 1.1 params.acc300 = 1.2
```

Comment:
TP bonus anchor according to https://forum.square-enix.com/ffxi/threads/42615 ramps up from 2k to 3k tp. Could probably be a guess of 1.05 to 1.2 on 2k/3k, but again not my choice alone.

Reply:
I have found more evidence that acc bonuses are not percents but static values. 10 acc at 500 acc has the same strength as 10 acc as 1000 anyway due to the formula being linear. this will have to be re-evaluated later but I am willing to let this slide for now.